sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -93,12 +93,8 @@ private static DigitalTwinsClient GetDigitalTwinsClient(string tenantId, string
 
             // By using the ClientSecretCredential, a specified application Id can login using a
```

Comment:
You'll need to update the comment, and comments in the readme, and probably in the ctor doc comments outside the example.

Reply:
ahhh yes. Will do it!

Reply:
Also no longer need to pass any arguments to the method.